operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

```diff
@@ -0,0 +1,475 @@
+        | wn01            | x3000             | u04                   | -     |              |      | j3                | sw-smn01          | x3000                  | u14                        | -     | j48                    |
+
+        `MgmtSwitchConnector` xname format: xXcCwWjJ
+
+        |   |                    | SHCD Column to reference   | Description
+        | - | ------------------ | -------------------------- | ----
+        | X | Cabinet number     | Destination Rack (R20)     | The Cabinet or rack number containing the management NCN.
+        | C | Chassis number     |                            | For air-cooled management switches the chassis is 0.
+        | W | Slot/Rack U        | Destination Location (S20) | The Slot/Rack U that the management switch occupies.
+        | J | Switch port number | Destination Port (U20)     | The switch port on the switch that the NCN BMC is cabled to.
+
+        ```bash
+        ncn-m# export MGMT_SWITCH_CONNECTOR=x3000c0w14j48
+        ```
+
+    1. **Skip if adding `ncn-m001`:** Determine the xname of the management switch by removing the trailing `jJ` from the `MgmtSwitchConnector` xname:
+
+        ```bash
+        ncn-m# export MGMT_SWITCH=x3000c0w14
+        ```
+
+    1. **Skip if adding `ncn-m001`:** Collect the BMC MAC address.
+        * If the NCN was previously in the system, recall the BMC MAC address recorded from the [Remove NCN Data](Remove_NCN_Data.md) procedure.
+
+        * Alternatively, view the MAC address table on the management switch that the BMC is cabled to.
+
+            1. Determine the alias of the management switch that the BMC is connected to:
+
+                ```bash
+                ncn-m# cray sls hardware describe $MGMT_SWITCH --format json | jq .ExtraProperties.Aliases[] -r
+                ```
+
+                Example output:
+
+                ```text
+                sw-leaf-bmc-001
+                ```
+
+            1. SSH into the management switch that the BMC is connected to:
+
+                ```bash
+                ncn-m# ssh admin@sw-leaf-bmc-001.hmn
+                ```
+
+            1. Locate the switch port that the BMC is connected to and record its MAC address. In the commands below, change the value of `1/1/39` to match the BMC switch port number (the BMC Switch port number is the `J` value in the in the `MgmtSwitchConnector` xname `xXwWjJ`).
+                For example, with the following `$MGMT_SWITCH_CONNECTOR` value:
+
+                ```bash
+                ncn-m# echo $MGMT_SWITCH_CONNECTOR
+                ```
+ 
+                Example output:
+
+                ```text
+                x3000c0w14j39
+                ```
+
+                The switch port number for the above `MgmtSwitchConnector` xname would be `39`, so use `1/1/39` instead of `1/1/48` in the commands below.
+
+                **Dell Management Switch**
```

Comment:
```suggestion
                - **Dell Management Switch**
```

Reply:
This and the related changes below resulted in a md style failure - I will need to rework this part
operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:126:1 MD004/ul-style Unordered list style [Expected: asterisk; Actual: dash]
operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:138:1 MD004/ul-style Unordered list style [Expected: asterisk; Actual: dash]

Reply:
I reverted this suggestion and the same for Aruba below due to md style error